data/publiccloud/azure_vmss.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash -u
+set -o pipefail
+. $(dirname "${BASH_SOURCE[0]}")/azure_lib_fn.sh
+#create a virtual machine scale set
+#list, show and delete the created resource
+#Report list of cli's used
+
+# Virtual Machine Scale Set AZURE CLI testing script
+
+#Required AZ setting variable block
+grpname="${1:-oqaclitest}"
+location="${2:-westus}"
+vmname="${3:-oqaclivmss}"
+ssh_key="${4:-oqaclitest-sshkey}"
+vmximagename="${5:-UbuntuLTS}"
+
+# Variable block
+account="Azure RD"
+admin="azureuser"
+policy="automatic"
```

Comment:
why this is a variable if it is used in single place ?

Reply:
done